db/migrate/20220128182721_add_url_submit_to_assignment_properties.rb

```diff
@@ -0,0 +1,5 @@
+class AddUrlSubmitToAssignmentProperties < ActiveRecord::Migration[6.1]
+  def change
+    add_column :assignment_properties, :url_submit, :boolean, default: false
```

Comment:
I always forget what the default is here but we don't want to allow nil values so add a `null: false` if the default is that it allows null values (or just add it to be safe if you're not sure)

Reply:
the default here is false (i.e. url submissions are disabled). I will still add the null: false just to be safe though